pkg/oc/cli/logs/logs.go

```diff
@@ -171,8 +209,9 @@ func (o *LogsOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args []st
 		}
 		o.Options = bopts
 
-	case appsapi.Resource("deploymentconfig"),
-		appsapi.Resource("deploymentconfigs"):
+	case appsapiv1.Resource("deploymentconfig"),
+		appsapiv1.Resource("deploymentconfigs"):
+		// TODO: switch to using appsapiv1.DeploymentLogOptions once originpolymorphichelpers.LogsForObjectFn supports appsv1
```

Comment:
@deads2k fyi. Using the external api here to create the DeploymentLogOptions object would also involve changes to the logsforobject Origin polymorphic helper: https://github.com/openshift/origin/blob/master/pkg/oc/originpolymorphichelpers/logsforobject.go#L33

Reply:
> @deads2k fyi. Using the external api here to create the DeploymentLogOptions object would also involve changes to the logsforobject Origin polymorphic helper:

update that in this pull..